src/components/CopyTextButton.tsx

```diff
@@ -8,29 +8,43 @@ import { CSSProperties, useContext, useState } from 'react'
 export default function CopyTextButton({
   value,
   style = {},
+  button,
```

Comment:
Addressing this comment: https://github.com/jbx-protocol/juice-interface/pull/2117#discussion_r986502348